apps/frontend-manage/src/components/questions/QuestionEditModal.tsx

```diff
@@ -929,8 +936,17 @@ function QuestionEditModal({
                           <Button
                             fluid
                             className={{
-                              root: 'font-bold border border-solid border-uzh-grey-100',
+                              root: twMerge(
+                                'font-bold border border-solid border-uzh-grey-100',
+                                values.type === QuestionType.Kprim &&
+                                  values.options.choices.length >= 4 &&
+                                  'opacity-50 cursor-not-allowed'
+                              ),
                             }}
+                            disabled={
```

Comment:
Should we maybe directly show 4 empty answer options for KPRIM questions as we do not allow less anyways at tue moment? Then we could hide the "add answer button" completely for this question type and maybe even simplify the validation?

Reply:
I understand your suggestion about directly displaying four empty answer options for KPRIM questions, especially since we don't allow fewer than that. It seems like a good way to enhance user experience by simplifying the interface.

However, implementing this change is proving to be more complex than initially thought:

1. Existing Bug: During the implementation, I discovered an existing bug, which gets triggered when:

- A user clicks on Add new answer.
- Then, deletes the top-most field.
- Upon attempting to modify the content of the newly added field, it inadvertently creates a new field on the click action. This behavior isn't new as I could verify it on the V3 branch.

2. The above behavior complicates KPRIM question management. Consider this scenario:

- A user creates 5 answer options for an SC question.
- They switch the question type to KPRIM. Now, all 5 answers are displayed, which goes against the KPRIM format.
- The user therefore needs to delete one answer as only 4 are allowed. However, if they try to remove the 4th field and then change the content of the last one (which was previously the 5th field), the system again creates a new field on click, even duplicating the content if present.

3. Preview Discrepancies: This issue has also caused discrepancies in the number of fields displayed in the preview. Addressing this is also planned for this PR.

Anyways, I'm currently working on addressing all of these issues :)